lib/src/base_request.dart

```diff
@@ -117,11 +118,18 @@ abstract class BaseRequest {
   /// the request is complete. If you're planning on making multiple requests to
   /// the same server, you should use a single [Client] for all of those
   /// requests.
-  Future<StreamedResponse> send() async {
+  ///
+  /// If [contentTimeout] is not null the request will be aborted if it takes
```

Comment:
I'd usually say "If [contentTimeout] is provided, the ...."
I know you can technically provide a `null` argument, but users will understand it anyway. After all, you talk about the duration afterwards, and `null` is not a duration.